cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -35,21 +38,27 @@ def report_menus(group, appliance):
 
 def crud_files_reports():
     result = []
-    if not report_crud_dir.exists:
-        report_crud_dir.mkdir()
-    for file_name in report_crud_dir.listdir():
+    if not REPORT_CRUD_DIR.exists:
```

Comment:
This data_path based object is a `LocalPath` type, and `exists` is a bound method, not a property. 

```
In [14]: from cfme.utils.path import data_path

In [15]: DIR = data_path.join('reports_crud')

In [16]: DIR.exists
Out[16]: <bound method LocalPath.exists of local('/home/mshriver/repos/integration_tests/data/reports_crud')>

In [17]: DIR.exists()
Out[17]: True

```

Additionally, LocalPath has a method to do what you're trying to do here, called `ensure`. It takes boolean kwarg `dir` for a directory vs a file.

```
In [19]: DIR = data_path.join('junk_path')

In [20]: DIR.exists()
Out[20]: False

In [21]: DIR.ensure(dir=True)
Out[21]: local('/home/mshriver/repos/integration_tests/data/junk_path')

In [22]: DIR.exists()
Out[22]: True
```

Reply:
Ah okay! I had copy-pasted the whole thing from another test file, didn't check it in details :/